elixir/apps/domain/lib/domain/auth.ex

```diff
@@ -459,13 +459,23 @@ defmodule Domain.Auth do
   end
 
   def create_identity(
```

Comment:
Just double-checking we have test coverage for this function.

Reply:
Yes, this function has the exact same functionality as before.  I just took the functionality from `create_identity/3` and moved it explicitly in to this function.  Here are the existing tests:
https://github.com/firezone/firezone/blob/1f5d28592b9eda1fd1b723122e3f773fa99355b7/elixir/apps/domain/test/domain/auth_test.exs#L2380

You are making me realize I should update one of the tests to verify that the `created_by_subject` is properly set though.  I'll get that updated.

Reply:
updated